docs/src/routes/button/demo.md

```diff
@@ -110,23 +113,88 @@ To trigger an operation.
     columns: ['Property', 'Description', 'Type', 'Default'],
     data: [
       {
-        property: 'color',
-        description: 'The color',
+        property: 'disabled',
+        description: 'The disabled state of button',
+        type: 'Boolean',
+        default: 'false'
+      },
+      {
+        property: 'ghost',
+        description: 'Makes the background transparent and inverts the text and border colors',
+        type: 'Boolean',
+        default: 'false'
+      },
+      {
+        property: 'href',
+        description: 'The url of link button',
         type: 'String',
-        default: 'white'
+        default: '-'
+      },
+      {
+        property: 'htmlType',
+        description: 'Set the original html type of button',
+        type: 'String',
+        default: 'button'
+      },
+      {
+        property: 'icon',
+        description: 'Set the icon component of button',
+        type: 'SvelteComponent',
+        default: '-'
+      },
+      {
+        property: 'loading',
+        description: 'Set the loading status of button',
+        type: 'Boolean | { delay: Number }',
+        default: 'false'
+      },
+      {
+        property: 'shape',
+        description: 'Can be set to circle, round or omitted',
+        type: 'Boolean | { delay: Number }',
+        default: 'false'
       },
       {
         property: 'size',
-        description: 'The main size',
-        type: 'Number',
-        default: '10'
+        description: 'Set the size of button',
+        type: 'large | middle | small',
```

Comment:
Instead of middle do you think we should use default? I think it’s more standard.